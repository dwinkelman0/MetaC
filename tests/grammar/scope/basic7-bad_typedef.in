{
    typedef struct { int x; int y; };
    const Coord_t *coord;
}