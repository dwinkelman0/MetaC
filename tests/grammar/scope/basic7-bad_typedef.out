{
    const Coord_t *coord;
}